react/w-books/.env.development

```diff
@@ -0,0 +1 @@
+REACT_APP_API_BASE_URL=https://books-training-rails.herokuapp.com/api/v1/
```

Comment:
Add this file to .gitignore file and then run **git rm --cached .env.development**

Reply:
ready